### ISSUE TITLE
Interface with different permission levels

### ISSUE THREAD

**Issue by Tacioandrade** [duplicati/duplicati#2901]
I am starting to put Duplicati in some places where besides me as administrator other analysts access duplicati to perform tasks like restore backups among other activities.

However with Duplicati as it currently works, when accessing the administration interface, the analyst or technician has access to sensitive information from backup such as: option to export the job, user and password of the used cloud service, etc.

So I thought, for a possible future version, a system where there are at least 2 types of user: administrator and support, where the support can only execute jobs like forcing the execution of a job or restoration of files?

I know it would not be something for a version as soon as and yes, something for a possible version 2.1 or higher, but is the idea valid for the project?

**Comment by kenkendk** [comment]
Yes, we have discussed this for a long time, but not found the time to implement it.
At least there should be a "view only" interface, possibly with the option to pause backups.

**Comment by Tacioandrade** [comment]
No doubt, I have a friend who showed interest in functionality also and has already programmed in C #, it may be that he is able to help in this matter, I will pass this post to him and maybe no other contributor appears to the source code of the project. Hehehe
